src/main/webapp/user-page.html

```diff
@@ -43,6 +43,7 @@ <h1 id="page-title">User Page</h1>
       <br/>
       <textarea name="text" id="message-input"></textarea>
       <br/>
+      <input value="" name="recipient" id="recipient-input">
```

Comment:
Let's add some kind of label here so that the user knows what this box is for, could be something like:

```html
<label for="recipient-input">To:</label>
```

before the `<input>`.

Reply:
Thanks! :) On it!